src/highlight/HighlightAnnotations.tsx

```diff
@@ -1,45 +1,170 @@
 import * as React from 'react';
-import HighlightCanvas from './HighlightCanvas';
+import noop from 'lodash/noop';
 import HighlightCreator from './HighlightCreator';
 import HighlightList from './HighlightList';
-import { AnnotationHighlight } from '../@types';
-import { isValidHighlight, sortHighlight } from './highlightUtil';
-
+import PopupReply from '../components/Popups/PopupReply';
+import SingleHighlightAnnotation from './SingleHighlightAnnotation';
+import { AnnotationHighlight, Rect } from '../@types';
+import { CreatorHighlight, CreatorStatus, Mode } from '../store';
 import './HighlightAnnotations.scss';
 
+type CreateArg = {
```

Comment:
Should this live in a `highlight/actions.ts` file, similar to how regions is structured?

Reply:
It will eventually -- this PR just doesn't have any of the store related code but this will eventually move there. this is just a temporary home